src/properties-minifier.ts

```diff
@@ -48,6 +48,8 @@ export class PropertiesMinifier {
 			return this.createNewAccessExpression(node, program);
 		} else if (ts.isBindingElement(node)) {
 			return this.createNewBindingElement(node, program);
+		} else if (isConstructorParameterRefernece(node, program)) {
```

Comment:
Typo in "Refernece", otherwise looks good. Thanks!

Reply:
thx, fixed the typo